draft-ietf-quic-tls.md

```diff
@@ -541,7 +541,7 @@ older than 1.3 is negotiated.
 
 QUIC requires that the initial handshake packet from a client fit within the
 payload of a single packet.  The size limits on QUIC packets mean that a record
-containing a ClientHello needs to fit within 1197 octets.
+containing a ClientHello needs to fit within 1165 octets.
```

Comment:
I'm OK with that.  As long as we stop changing the overhead when we ship the spec.